CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## dbt 0.17.1 (Release TBD)
 
+### Fixes
+- Hash name of local packages
```

Comment:
Please link to this PR as well, otherwise lgtm

Reply:
ha, but how does one know the PR number before creating the PR 🤔

Reply:
Two ways: I look at the highest existing PR/issue numbers and add 1, and then if someone opens one in the interim I fix it with a `git commit --amend; git push -f`. Alternatively, I open it as a draft, copy the issue number in to the changelog, and do the `git commit --amend; git push -f` before I ask for reviews. Both suck!